lib/util.js

```diff
@@ -66,9 +66,11 @@ const Util = module.exports = {
       // if there are conflicts, log debug output
       if (conflicts.length > 0) {
         _.each(conflicts, conflictKey => {
-          app.log.debug('The config setting', `${key}.${conflictKey}`, 'is defined by both',
-            packName, 'and the application. The', conflictKey,
-            'config defined in', packName, 'will be ignored.')
+          if (app.config[key] && _.isArray(app.config[key][conflictKey])){
```

Comment:
Sure, that makes sense. I would clarify slightly to say which object is merged into which. Because if there are keys in common, the destination keys will be overwritten.